packages/core/src/components/footer/footer.tsx

```diff
@@ -30,10 +30,12 @@ export class TdsFooter {
         <footer>
           {usesTopSlot && <slot name="top"></slot>}
           <div class="footer-main">
-            <div class="footer-main-top">
-              {usesStartSlot && <slot name="start"></slot>}
-              {usesEndSlot && <slot name="end"></slot>}
-            </div>
+            { (usesStartSlot || usesEndSlot) && (
+              <div class="footer-main-top">
+                {usesStartSlot && <slot name="start"></slot>}
+                {usesEndSlot && <slot name="end"></slot>}
+              </div>
+            )}
```

Comment:
Not sure how you want to tackle this @theJohnnyMe, but I'd say this could be considered a breaking change since it could change the layout of applications.